public/css/style.css

```diff
@@ -0,0 +1,83 @@
+.container {
+  margin: 0 auto;
+  border-width: 14px;
+  border-style: solid;
+  border-color: pink;
+  border-radius: 95px;
+  text-align: center;
+  width: 600px;
+  height: 775px;
+  background-color: lightblue;
+  font-family: 'Comfortaa', cursive;
+  color: black;
+}
+
+.secret-word {
+  font-family: 'Inconsolata', monospace;
+  font-size: 40px;
+  font-weight: bold;
+  padding-top: 15px;
+}
+
+.link {
+  border-radius: 12px;
+  background-color: pink;
+  padding-top: 25px;
+  width: 110px;
+  height: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 45px;
+  margin-top: 5px;
+  line-height: 10px;
+  padding: 15px;
+  position: relative;
+  top: 20px;
+}
+
+a {
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+  border-bottom: 1px solid black;
+  padding-bottom: 2px;
+}
+
+a:hover {
+  color: white;
+  font-family: 'Open Sans';
+  font-size: 20px;
+  border-bottom: 1px solid white;
+}
+
+.wrong-guesses {
+  font-size: 18px;
+  padding-top: 20px;
+}
+
+img {
+  height: 300px;
+  width: 300px;
+}
+
+.button {
```

Comment:
I'd suggest a `cursor: pointer` for every button. I think it's strange if the mouse cursor is still an arrow when you hover over a button.